package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@metamask/bip39": "^4.0.0",
     "@metamask/eth-hd-keyring": "^4.0.2",
-    "browser-passworder": "MetaMask/browser-passworder#e66f3ca0b57f4ef9de4acd782e92b850cb0fe6bb",
+    "browser-passworder": "darkwing/browser-passworder#mv3-login-dist",
```

Comment:
Before merging this PR, those changes should ideally be merged to `MetaMask/browser-passworder`. If there is a blocker for this, let's make it a branch of the upstream repo and point at the commit hash? 

The reasoning is to not have builds produce different results depending on the current state of branch in an individual's external repo.

Reply:
This has just been published to npm as v4.0.1